ADT/sftp_oag_client_maytech.py

```diff
@@ -151,11 +164,27 @@ def main():
     oaghistory.close()
     logger.info("Downloaded %s files", downloadcount)
 
-    if downloadcount == 0:
-        status=-1
 
-    logger.info("Done Status %s", status)
-    print status
+# Move files to S3
+    logger.info("Starting to move files to S3")
+    processed_oag_file_list = [f for f in os.listdir(DOWNLOAD_DIR)]
+    if processed_oag_file_list:
+        for filename in processed_oag_file_list:
+            boto_s3_session = boto3.Session(
```

Comment:
Creating a new boto session per file would seem inefficient, versus starting the session before the `for` loop.

Reply:
Corrected.